lightning/src/blinded_path/mod.rs

```diff
@@ -78,22 +79,30 @@ impl BlindedPath {
 	/// Create a blinded path for a payment, to be forwarded along `path`. The last node
 	/// in `path` will be the destination node.
 	///
-	/// Errors if `path` is empty or a node id in `path` is invalid.
+	/// Errors if:
+	/// * `path` is empty
+	/// * a node id in `path` is invalid
```

Comment:
pre-existing: `path` is no longer a name of a parameter?

Reply:
Also where does it error if `path` (`intermediate_nodes`?) is empty?

Reply:
> Also where does it error if `path` (`intermediate_nodes`?) is empty?

We allow creating 1-hop blinded routes so shouldn't need to error there! `payee_tlvs` will always be present.